src/luv.c

```diff
@@ -242,6 +242,13 @@ static const luaL_Reg luv_functions[] = {
   {"version", luv_version},
   {"version_string", luv_version_string},
 
+  // thread.c
+  {"new_thread", luv_new_thread},
+  {"thread_create", luv_thread_create}, 
+  {"thread_equal", luv_thread_equal},
+  {"thread_self", luv_thread_self},
+  {"thread_join", luv_thread_join},
+
```

Comment:
We should also add the method form for `:create`, `:equal`, and `:join` in the metatable for threads so you can do:

```lua
uv.new_thread():create(function () 
  -- ...
end)
```

Also, is there a reason to have `uv.thread_create` be apart from `uv.new_thread`?